### PR TITLE
Map legend restyling

### PR DESCRIPTION
**Pull request by smacmullan**
Change the legend labels. Add sass npm package to project, which automatically compiles scss files to css when the project builds.

### CODE REVIEW

**Comment by lauriemerrell** [comment]
Just adding a comment I made in Slack for visibility -- I love the legend phrasing and reordering the colors, but I *think* we may want to stick with a blue/orange color scale rather than blue/red for better accessibility for colorblind users. I would defer to @amy-corson though who has done more research on this!

File: src/css/main.css [hunk 1]
```diff
@@ -689,30 +689,39 @@ header .h1-container .subtitle-container img {
...
-  background-color: #FFED39;
+  background-color: #1A85FF;
 }
 
+.filter-key h4{
+  text-align: center;
+}
+.percentile-text{
+  text-align: right;
+}
```
Comment (amy-corson) [inline src/css/main.css:722]: per above note, if we keep the `flex` over the `grid`, just add a `width: 100%` here to get the `text-align` to work!

**Review by amy-corson — CHANGES_REQUESTED** [review]
Hello! First of all, thank you SO MUCH for taking the initiative to address this! I really love the labeling changes in the key, and I agree strongly that this language is so much easier to understand! There are two things I'd like to discuss/address in this PR:
- I personally would want to see the colors reverted to the yellow/purple heatmap, because I think i value the color contrast over trying to assign colors with more positive/negative connotations. Open to discussing this more, I know we've got a Slack thread going.
- This project uses an SCSS compiler, so ideally you would be making these changes in the SCSS files not editing directly, while also using an extension such as live sass compiler! (this is such a terrible system we need to update it.)

**Comment by smacmullan** [comment]
I switched to a yellow-purple gradient. I'm not sure if you were asking for the original color scheme? In the Slack thread we were discussing use a two-color gradient because it's easier to interpret hot and cold on the heat map.

**Comment by smacmullan** [comment]
I got sass working. If you make changes to scss files while running "npm start", changes will auto-update.

**Comment by amy-corson** [comment]
@smacmullan Can you update the PR description to outline both of the packages you're adding, what they do, and why they're necessary?

**Comment by smacmullan** [comment]
I changed the description. Unless I'm missing something, I've only add the one package: sass.

**Comment by smacmullan** [comment]
Immutable is a sass dependency (shows up in the package-lock).

**Review by amy-corson — APPROVED** [review]
LGTM! Thanks so much!

**Pull request merged**